vacasa/connect/connect.py

```diff
@@ -128,6 +128,65 @@ def get(self, uri, params: dict = None):
 
         return r.json()
 
+    """ Create a unit via connect. Required args are at the top of the list.
+        https://vacasa.docs.stoplight.io/units/postv1units
+    """
+    def create_unit(self,
+                    region_id: int,
+                    turnover_day: int,
+                    housing_type: str,
+                    code: str= None,
```

Comment:
i think we just need a `pep8` autoformatter pass to tweak the whitespace on all of these
```python
def example(param1=None,
            param2: str = None):
    pass
```

Reply:
We should also move the docstring inside the method to match the convention of this SDK.